src/store/users/handlers.ts

```diff
@@ -0,0 +1,11 @@
+import { Dispatch } from 'redux';
+
+import usersAction from './slice';
+import { getUsersApi } from './api';
+
+type IHandleGetUsersAction = () => (dispatch: Dispatch) => void;
```

Comment:
you can create a type, that returns (dispatch: Dispatch) => void; and use it in future